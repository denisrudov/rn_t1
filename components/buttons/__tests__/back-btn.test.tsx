import renderer from 'react-test-renderer'
import BackBtn from '@/components/buttons/back-btn'


describe('buttons', () => {
    it('should render', () => {
        const tree = renderer.create(<BackBtn/>).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
