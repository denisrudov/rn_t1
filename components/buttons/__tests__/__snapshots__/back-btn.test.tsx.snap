// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` should render 1`] = `
<View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <RNSVGSvgView
      align="xMidYMid"
      bbHeight="24"
      bbWidth="24"
      fill="none"
      focusable={false}
      height={24}
      meetOrSlice={0}
      minX={0}
      minY={0}
      stroke="#fff"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeWidth={2}
      style={
        [
          {
            "backgroundColor": "transparent",
            "borderWidth": 0,
          },
          {
            "flex": 0,
            "height": 24,
            "width": 24,
          },
        ]
      }
      vbHeight={24}
      vbWidth={24}
      width={24}
      xmlns="http://www.w3.org/2000/svg"
    >
      <RNSVGGroup
        fill={null}
        propList={
          [
            "fill",
            "stroke",
            "strokeWidth",
            "strokeLinecap",
            "strokeLinejoin",
          ]
        }
        stroke={
          {
            "payload": 4294967295,
            "type": 0,
          }
        }
        strokeLinecap={1}
        strokeLinejoin={1}
        strokeWidth="2"
      >
        <RNSVGPath
          d="m15 18-6-6 6-6"
          fill={null}
          propList={
            [
              "fill",
              "stroke",
              "strokeWidth",
              "strokeLinecap",
              "strokeLinejoin",
            ]
          }
          stroke={
            {
              "payload": 4294967295,
              "type": 0,
            }
          }
          strokeLinecap={1}
          strokeLinejoin={1}
          strokeWidth="2"
        />
      </RNSVGGroup>
    </RNSVGSvgView>
  </View>
</View>
`;
